atmosphere/index.html

```diff
@@ -0,0 +1,202 @@
+      </section>
+    </article>
+    <section class="poster-container">
+      <div class="card">
+        <header class="card-header">
+          <p>CONSEQUAT</p>
+          <h2>Fusce lobortis lorem semper sagittis</h2>
+          <hr class="horizontal">
+        </header>
+        <img src="assets/atmosphere0.jpeg" alt="image" class="atmosphere-img">
+        <div class="card-content">
+          <p>
+            Phasellus in ante vitae nibh porttitor tempus vitae ante. Lorem
+            blandit dolor elementum dolor. Sed quat diam, aliquet tempus felis.
+            Phasellus et magna vitae sed porttitor tempus vitae ut ante lorem
+            blandit.
+          </p>
+          <a href="#" class="btn-more">More</a>
+        </div>
+      </div>
+    </section>
+    <hr />
+    <article class="card-container bottom">
+      <header class="card-header">
+        <p>malesuada</p>
+        <h2>Aliquam tincidunt mauris eu risus</h2>
+      </header>
+      <section class="card-content">
+        <p class="para">09.04.16</p>
+        <h3 class="card-head">MAGNA ETIAM SED LOREM IPSUM DOLOR AMET</h3>
+        <img src="assets/atmosphere01.jpeg" alt="image"  class="atmosphere-img"/> <br />
+        <p>
+          Phasellus in ante vitae nibh porttitor tempus vitae ut ante.
+          Vestibulum blandit dolor elementum viverra. Sed quat a diam, aliquet
+          tempus felis. Phasellus et magna vitae nibh porttitor tempus vitae.
+        </p>
+        <a href="#" class="btn-more">More</a>
+      </section>
+      <section class="card-content">
+        <p class="para">08.28.16</p>
+        <h3 class="card-head">SED FEUGIAT ET MAURIS ADIPISCING DOLOR</h3>
+        <img src="assets/atmosphere02.jpeg" alt="image"  class="atmosphere-img"/> 
+        <p>
+          Phasellus in ante vitae nibh porttitor tempus vitae ut ante.
+          Vestibulum blandit dolor elementum viverra. Sed quat a diam, aliquet
+          tempus felis. Phasellus et magna vitae nibh porttitor tempus vitae.
+        </p>
+        
+        <a href="#" class="btn-more">More</a>
+      </section>
+      <section class="card-content">
+        <p class="para">08.22.16</p>
+        <h3 class="card-head">TELLUS MAGNA DOLORE SED ELEMENTUM</h3>
+        <img src="assets/atmosphere03.jpeg" alt="image"  class="atmosphere-img"/> 
+        <p>
+          Phasellus in ante vitae nibh porttitor tempus vitae ut ante.
+          Vestibulum blandit dolor elementum viverra. Sed quat a diam, aliquet
+          tempus felis. Phasellus et magna vitae nibh porttitor tempus vitae.
+        </p>
+        
+        <a href="#" class="btn-more">More</a>
+      </section>
+    </article>
+    <hr />
+    <article class="contact-section">
+      <section class="Address">
+        <ul class="address-content">
+          <li>
+            <i class="fa-solid fa-house"></i>
+          </li>
+          <li>
+            <h4>Address</h4>
+            <p>1234 Fictional Avenue  <p>Nashville, TN 00000</p></p>
```

Comment:
avoid nested p, instead use any inline elements . P does not allow block elements